.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "golang build env",
```

Comment:
Looks like these `devcontainer` files were added to the commit, you may want to update the `.gitignore`